docs/architecture.rst

```diff
@@ -0,0 +1,56 @@
+Node Architecture
+=================
+
+Each node consists of 2 main logical components:
+
+- `Journal <https://sawtooth.hyperledger.org/docs/core/releases/latest/architecture/journal.html>`_ - responsible for consensus mechanism, storage and networking.
```

Comment:
It is always better not to use pics from 3rd parties (even though they are secured via https)

Reply:
Do you mean to use the third party sich as imgur? Because Sawtooth is currently a "third-party"

Reply:
ok, will move out all pics